scripts/install

```diff
@@ -156,6 +147,17 @@ ynh_print_info "Configuring log rotation..."
 touch "$final_path/data/log.txt"
 ynh_use_logrotate "$final_path/data/log.txt"
 
+#=================================================
+# SECURE FILES AND DIRECTORIES
+#=================================================
+# set proper permissions
+ynh_print_info "Set permissions..."
+find $final_path -type f | xargs chmod 644
+find $final_path -type d | xargs chmod 755
```

Comment:
Uh I have no idea what I'm talking about but is it really what we wanna do here ?

Is it okay that *all users* on the system will be able to read every file in /var/www/$app ? Are we sure that there's absolutely no info that should stay private in that folder ?

Reply:
Good point, I don't see any reason to do that.